termui/bug_table.go

```diff
@@ -290,10 +290,19 @@ func (bt *bugTable) render(v *gocui.View, maxX int) {
 	columnWidths := bt.getColumnWidths(maxX)
 
 	for _, excerpt := range bt.excerpts {
-		summaryTxt := fmt.Sprintf("C:%-2d L:%-2d",
-			excerpt.LenComments,
-			len(excerpt.Labels),
-		)
+		summaryTxt := fmt.Sprintf("%4d 💬", excerpt.LenComments)
+		labelsTxt := "" // fmt.Sprintf("L:%-2d", len(excerpt.Labels))
+
+		nbLabels := 0
+		for _, l := range excerpt.Labels {
+			lc := l.Color()
+			lc256 := lc.Term256()
+			labelsTxt += lc256.Escape() + " ◼" + lc256.Unescape()
+			nbLabels++
+			if nbLabels >= 5 {
```

Comment:
It would be nice to use `…` to signal that there is more than what is displayed.

Reply:
Yep, it's fixed in my next commit!